shynet/dashboard/templates/base.html

```diff
@@ -17,23 +17,23 @@
   {% endblock %}
 </head>
 
-<body class="bg-neutral-100 min-h-full">
+<body class="bg-neutral-100 min-h-full overflow-x-hidden">
   {% block body %}
 
   <section class="max-w-screen-xl mx-auto px-4 py-4 md:py-12 md:flex">
     <aside
-      class="mb-8 md:w-2/12 md:pr-6 relative flex flex-wrap md:block justify-between items-center overflow-x-hidden">
+      class="mb-2 md:w-2/12 md:pr-6 relative flex flex-wrap md:block justify-between items-center overflow-x-hidden">
       <a class="icon ~urge ml-2 md:ml-6 md:mb-8 md:mt-3" href="{% url 'dashboard:dashboard' %}">
         <i class="fas fa-binoculars fa-3x text-urge-600 hidden md:block"></i>
         <i class="fas fa-binoculars fa-2x text-urge-600 md:hidden"></i>
       </a>
 
-      <button class="button ~neutral !low md:hidden"
+      <a class="button ~neutral !low md:hidden"
```

Comment:
We should probably also add `role="button"` to the `<a>` tag to keep this semantic.

Reply:
Reason for this change was because the outline stayed visible after toggling, which I found a bit ugly. I should find better solution so it is still focusable by keyboard aswell